packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -431,6 +432,23 @@ export async function getChatAuthState(cwAuth = AuthUtil.instance): Promise<Feat
     return state
 }
 
+export function isPreviousQUser() {
```

Comment:
Probably worth a docstring that says what this is for. This eventually won't be needed, right?

    /** Returns true if the user is an existing Toolkit user who has newly installed the Q standalone extension. */